src/main/java/hudson/plugins/collabnet/orchestrate/BuildNotifier.java

```diff
@@ -392,13 +389,14 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
         PrintStream consoleLogger = listener.getLogger();
 
         if(getSupportEventQ() == true && getSupportWebhook() == true){
-            Helper.markUnstable(
-                    build,
-                    consoleLogger,
-                    "Build information NOT sent: Can't send notification to TeamForge and EventQ at the same" +
-                            " time." +
-                            "Please use either TeamForge/EventQ.", getClass().getName());
-            return true;
+            setSupportEventQ(false);
+            RadioConfig config = getConfig();
+            config.setWebhookUrl(Helper.getWebhookUrl(getCtfUrl()));
```

Comment:
Yes, there is a case where webhook url is already available in the config, but it better to get the recent URL. Say for instance user might be using 19.1 CTF release where WEBR is of v2 engine, when the user move to new release WEBR v4 engine, there is a change in url path. To avoid manually changing the config again, we are getting the updated url while pushing the notification without changing any configuration.

Reply:
What if user is still on CTF 19.1?